src/Client/Shared/BaseLayout.razor.cs

```diff
@@ -22,7 +22,8 @@ protected override async Task OnInitializedAsync()
         _themePreference = await _clientPreferenceManager.GetPreference() as ClientPreference;
         if (_themePreference == null) _themePreference = new ClientPreference();
         SetCurrentTheme(_themePreference);
-        _interceptor.RegisterEvent();
+
+        // _interceptor.RegisterEvent();
```

Comment:
interceptors are removed?

Reply:
I'm not sure we need it... What exactly are you using it for? I think everything we neeed to do wrt the httpclient (requesting access and refresh tokens) should be possible using only the messagehandler (the remoteauthenticatorview also does it that way).

Reply:
https://github.com/fullstackhero/blazor-wasm-boilerplate/blob/main/src/Client.Infrastructure/Services/Interceptor/HttpInterceptorService.cs

It was used to logout and throw a snackbar message when the API returns a 401 status code. What happens now when the jwtoken expires?

Reply:
Yes but that seems something temporary as I would think you would refresh the access token behind the scenes (using the httpmessagehandler) so the token doesn't expire... also the redirecting (in case a new login is required when e.g. the refresh token is actually expired) should be possible from there. I know that's how the built-in remote authentication works. There you only need to register that AuthorizationMessageHandler which takes care of all that behind the scenes.